### ISSUE TITLE
A namespace cannot directly contain members such as fields or methods

### ISSUE THREAD

**Issue by bdelaney** [erichexter/twitter.bootstrap.mvc#79]
This error on build out of the Nuget Templates package, ControllerWithContext.tt showing a .cs output and that is source of error.
Used latest Nuget packages (all 3) in an MVC4 project, Empty template.
This occurred after the recommended project close and reopen.

Deleted the ControllerWithContext.cs output file and build works.

After installing the 3 NuGet packages, Package Manager GUI shows 4 packages, all are 1.0.90 version. Makes sense as I look at packages.config file and note what has been installed:
  <package id="Twitter.Bootstrap" version="2.3.1" targetFramework="net45" />
  <package id="twitter.bootstrap.mvc4" version="1.0.90" targetFramework="net45" />
  <package id="twitter.bootstrap.mvc4.htmlhelpers" version="1.0.90" targetFramework="net45" />
  <package id="twitter.bootstrap.mvc4.sample" version="1.0.90" targetFramework="net45" />
  <package id="twitter.bootstrap.mvc4.templates" version="1.0.90" targetFramework="net45" />

Looking forward to checking it out further.

**Comment by serra** [comment]
There seems to be missing some relevant information from your bug report. What has been installed?

**Comment by MilkyWayJoe** [comment]
I'm having the exact same issue.

I am using the MVC4 Internet Application Template.

I've installed the packages in this order:

PM> install-package twitter.bootstrap.mvc4
PM> install-package twitter.bootstrap.mvc4.sample
PM> install-package twitter.bootstrap.mvc4.templates
PM> install-package twitter.bootstrap.mvc4.templates.htmlhelpers

By the time it installed the "templates" I could see 3 items in my errors list:
![Untitled](https://f.cloud.github.com/assets/1315761/238488/9458e276-8839-11e2-91aa-4dd75fe7a747.png)

I have installed the HTML Helpers with no problems

**Comment by serra** [comment]
Hmm, I'll try this out. By `twitter.bootstrap.mvc4.templates.htmlhelpers`, I assume you mean `twitter.bootstrap.mvc4.htmlhelpers`? Note that the htmlhelpers package already should be installed with the sample too.

**Comment by serra** [comment]
This is because the custom template of `CodeTemplates\AddController\ControllerWithContext.tt` is not cleared.

Workaround for now ( @bdelaney and @MilkyWayJoe ) is to simply clear the Custom Tool in the file property window.

This is a valid bug report though - afaik the Custom Tool property was automatically cleared in earlier releases.

**Comment by serra** [comment]
Correction: I fixed this as part of #29, but that is pull request is still open. @erichexter or @MisterJames please consider merging #29.

**Comment by erichexter** [comment]
Im traveling and will be iffline foe the next two weeks.

James can you apply the pr?

**Comment by MilkyWayJoe** [comment]
@serra yeah, I wrote it incorrectly here, but I've actually installed `twitter.bootstrap.mvc4.htmlhelpers`
